applications/easysocial/pushnotify/pushnotify.php

```diff
@@ -359,7 +359,7 @@ public function onNotificationBeforeCreate( $item_dt)
 					$body['aps'] = array(
 						'alert' => $message,
 						'sound' => 'default',
-						"badge" => $count,
```

Comment:
Will it always be +1 ? No possibility of sending multiple notifications at once ?

Reply:
For every new action, EasySocial call the trigger to send notification. So there is no possibility of sending multiple notifications at once.